Cython/Plex/Actions.py

```diff
@@ -52,26 +52,6 @@ def same_as(self, other):
         return isinstance(other, Call) and self.function is other.function
 
 
-class Begin(Action):
```

Comment:
We have custom user `State`s in the scanner, so this looks like functionality that might be worth keeping in Plex, just in case.

Reply:
Done.